lib/cdo/data/logging/timed_task_with_logging.rb

```diff
@@ -8,9 +8,12 @@ def execute(args = nil)
       logger = RakeTaskEventLogger.new(self)
       logger.start_task_logging
       begin
+        RakeTaskEventLogger.increase_depth
```

Comment:
Could we use the Ruby `alias` or `alias_method` features to redefine `Rake::Task.execute` to do what it normally does but include the logging before/after?

Reply:
I'll work on this after I learned how to do the alias (added to my to do list)